tutorials/evoked/plot_20_visualize_evoked.py

```diff
@@ -0,0 +1,249 @@
+for e in evokeds_list:
+    print(e.comment)
+
+###############################################################################
+# To make our life easier, let's convert that list of :class:`~mne.Evoked`
+# objects into a :class:`dictionary <dict>` (make sure the order is correct!).
+# We'll use ``/``-separated dictionary keys to encode the conditions, like is
+# often done when epoching, because some of the plotting methods can take
+# advantage of that style of coding.
+
+conds = ('aud/left', 'aud/right', 'vis/left', 'vis/right')
+evks = dict(zip(conds, evokeds_list))
+
+###############################################################################
+# Plotting signal traces
+# ^^^^^^^^^^^^^^^^^^^^^^
+#
+# The most basic plot of :class:`~mne.Evoked` objects is a butterfly plot of
+# each channel type, generated by the :meth:`evoked.plot() <mne.Evoked.plot>`
+# method. By default, channels marked as "bad" are suppressed, but you can
+# control this by passing an empty :class:`list` to the ``exclude`` parameter
+# (default is ``exclude='bads'``):
+
+evks['aud/left'].plot(exclude=[])
+
+###############################################################################
+# Notice the completely flat EEG channel and the noisy gradiometer channel
+# plotted in red color. Like many MNE-Python plotting functions,
+# :meth:`evoked.plot() <mne.Evoked.plot>` has a ``picks`` parameter that can
+# select channels to plot by name, index, or type. In the next plot we'll show
+# only magnetometer channels, and also color-code the channel traces by their
+# location by passing ``spatial_colors=True``. Finally, we'll superimpose a
+# trace of the :term:`global field power <GFP>` across channels:
+
+evks['aud/left'].plot(picks='mag', spatial_colors=True, gfp=True)
+
+###############################################################################
+# Plotting scalp topographies
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# In an interactive session, the butterfly plots seen above can be
+# click-dragged to select a time region, which will pop up a map of the average
+# field distribution over the scalp for the selected time span. You can also
+# generate scalp topographies at specific times or time spans using the
+# :meth:`~mne.Evoked.plot_topomap` method:
+
+times = np.linspace(0.05, 0.13, 5)
+evks['aud/left'].plot_topomap(ch_type='mag', times=times, colorbar=True)
+
+###############################################################################
+
+fig = evks['aud/left'].plot_topomap(ch_type='mag', times=0.09, average=0.1)
+fig.text(0.5, 0.05, 'average from 40-140 ms', ha='center')
+
+###############################################################################
+# Additional examples of plotting scalp topographies can be found in
+# :ref:`ex-evoked-topomap`.
+#
+#
+# Current flow maps
+# ^^^^^^^^^^^^^^^^^
+#
+# Scalp topographies at a given time point can be augmented with arrows to show
+# the estimated direction of current flow, using the function
```

Comment:
it's not current flow but orientation of the magnetic field which is a vector field.